pkg/tsdb/cloudmonitoring/test-data/4-series-response-distribution-explicit.json

```diff
@@ -103,7 +103,16 @@
                 "4",
                 "7",
                 "12",
-                "8"
+                "8",
+                "0",
+                "0",
+                "0",
+                "0",
+                "0",
+                "0",
+                "0",
+                "0",
+                "0"
```

Comment:
this does not really represent reallity since in `BucketCounts` if the rest of values are 0, it's just ommited for performance reasons (that's why usually BucketCounts.length is smaller than the ExplicitBounds) but it's helpful to represent the issue in a unit test.